students/uss_tomasz/lesson_08_regular_expressions/8_2/strong_password_detection.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+
+def is_password_strong(password):
+    length_regex = re.compile(r'[a-zA-Z0-9]{8,}')
+    if len(re.findall(length_regex, password)) == 0:
```

Comment:
No need to use `findall`, you only need to check whether the condition is fulfilled. `search` is enough. Please correct  in this and other files in this PR